bfabric/bfabric2.py

```diff
@@ -75,19 +75,22 @@ def __init__(self, authClass, configClass, engine: BFABRIC_API_ENGINE = BFABRIC_
         else:
             raise ValueError("Unexpected engine", BFABRIC_API_ENGINE)
 
-    def read(self, endpoint: str, obj: dict, page: int = 1, plain: bool = False, **kwargs) -> ResultContainer:
+    # TODO: Perform pagination. Return inner values, i.e. val[endpoint].
+    def read(self, endpoint: str, obj: dict, page: int = 1, **kwargs) -> ResultContainer:
         results = self.engine.read(endpoint, obj, page = page, **kwargs)
         return ResultContainer(results, self.resultType)
 
-
+    # TODO: Perform pagination. Return inner values, i.e. val[endpoint].
     def readid(self, endpoint: str, obj: dict, page: int = 1, plain: bool = False, **kwargs) -> ResultContainer:
```

Comment:
```suggestion
    def readid(self, endpoint: str, obj: dict, page: int = 1, **kwargs) -> ResultContainer:
```

Reply:
I think that plain method would apply to the pagination within the wrapper, not within the engine. In my understanding, **kwargs would only contain keyword arguments that are specific to each engine, if any.